src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java

```diff
@@ -288,7 +314,7 @@ private void readHeader() throws IOException {
          * @throws IOException
          */
         public WFIterator() throws IOException {
-            final InputStream inputStream = ParsingUtils.openInputStream(path);
+            final InputStream inputStream = ParsingUtils.openInputStream(path, wrapper);
```

Comment:
It looks like the wrapping happens before unzipping here -- we should make it clear to the client that they are always wrapping raw unmodified byte streams (and verify that that's the case everywhere).